firebase-perf/src/main/java/com/google/firebase/perf/metrics/validator/PerfMetricValidator.java

```diff
@@ -140,28 +140,30 @@ public static String validateMetricName(@Nullable String str) {
    *     it can't be used.
    */
   @Nullable
-  public static String validateAttribute(@NonNull Map.Entry<String, String> attribute) {
+  public static void validateAttribute(@NonNull Map.Entry<String, String> attribute) {
     String key = attribute.getKey();
     String value = attribute.getValue();
-    if (key == null) {
-      return "Attribute key must not be null";
-    } else if (value == null) {
-      return "Attribute value must not be null";
+    if (key == null || key.length() == 0) {
+      throw new IllegalArgumentException("Attribute key must not be null or empty");
+    } else if (value == null || value.length() == 0) {
```

Comment:
The `else if` are unnecessarily when you are already throwing/returning for each condition. You can just use `if`.

Reply:
Ha! Obvious! Thanks for the reminder.